src/main/java/org/myrobotlab/service/abstracts/AbstractServo.java

```diff
@@ -713,32 +715,32 @@ public String publishServoDisable(ServoControl sc) {
              * info - sending whole servo is excessive
              */
   public String publishServoEnable(ServoControl sc) {
-    log.warn("publishServoEnable(name)");
+    log.info("publishServoEnable(name)");
     return sc.getName();
   }
 
   // TODO: why do we need this method here , invoke message cache misses
   // otherwise.
   public String publishServoEnable(AbstractServo sc) {
-    log.warn("publishServoEnable(servo)");
+    log.info("publishServoEnable(servo)");
     return publishServoEnable((ServoControl) sc);
   }
 
   @Override
-  public ServoControl publishServoMoveTo(ServoControl sc) {
-    log.warn("publishServoMoveTo(servo)");
-    return sc;
+  public ServoMove publishServoMoveTo(Double pos) {
+    log.info("publishServoMoveTo({})");
+    return new ServoMove(getName(), pos);
   }
 
   @Override
   public ServoSpeed publishServoSetSpeed(ServoControl sc) {
-    log.warn("publishServoSetSpeed(servo)");
+    log.info("publishServoSetSpeed(servo)");
     return new ServoSpeed(sc.getName(), sc.getSpeed());
   }
 
   @Override
   public ServoControl publishServoStop(ServoControl sc) {
-    log.warn("publishServoStop(servo)");
+    log.info("publishServoStop(servo)");
```

Comment:
should probably log  servo.getName()   and subsitute it via {} in the log message.